packages/orbit-components/src/SkipLink/SkipLink.stories.tsx

```diff
@@ -62,10 +62,12 @@ export const Playground = () => {
     "buttonLabel",
     "Description for screen readers about where the user currently is or describing a more complex component.",
   );
+  const dataTest = text("dataTest", "test");
+  const id = text("id", "ID");
```

Comment:
We tend to avoid adding knobs of props that have no effect. It's not a hard request, so I won't ask you to remove them, but if they don't anything to the story, there's no need to have them there 🙂

Reply:
Got you. No problem, I’ll remove them from my open MRs.

Reply:
Looks like I haven’t added them elsewhere so it should be good.